macos-guest-virtualbox.sh

```diff
@@ -320,6 +356,18 @@ if [[ -z "$(dmg2img -d 2>/dev/null)" ]]; then
     }
 fi
 
+# get CPU Information from /proc/cpuinfo
+modelname=$(cat /proc/cpuinfo | grep 'model name' | head -1)
+# check if cpu_profile contains something other than host, as to not disturb testers trying a different cpu_profile.
+if [[ ${cpu_profile} != "host" ]]; then
+	echo "cpu_profile Is $cpu_profile, Other Than host, So Not Changing Anything"
+elif grep Intel <<<"$modelname" > /dev/null; then
+	echo "Intel CPU Detected, So Not Changing Anything"
+elif grep "AMD" <<<"$modelname" > /dev/null; then
+	echo "AMD CPU Detected, So Changing cpu_profile To: Intel Core i7-3960X"
```

Comment:
As stated elsewhere, I will not maintain a database of CPUs and their matching profiles.